FEM/rf_msp_new.cpp

```diff
@@ -45,6 +45,8 @@
 #include "minkley.h"
 #include "burgers.h"
 
+#include "../MathLib/LinAlg/GaussAlgorithm.h"
```

Comment:
I think `../` is unnecessary

Reply:
By amending the corresponding commit, it is changed to
```
#include "LinAlg/GaussAlgorithm.h"
```
because that MathLib is included in CMakeLists.txt of FEM.

The one in testGaussAlgorithm is changed too.